packages/app/src/cli/services/dev/select-store.ts

```diff
@@ -39,9 +39,11 @@ export async function selectStore(
   org: Organization,
   developerPlatformClient: DeveloperPlatformClient,
 ): Promise<OrganizationStore> {
+  const clientName = developerPlatformClient.clientName
```

Comment:
should we use the available `clientName` field, or create a new one? I think `clientName` was created for the purposes of debugging.. then again, will we always have both clients?